changelog/fix_an_error_for_sole_nested_conditional.md

```diff
@@ -0,0 +1 @@
+* [#9448](https://github.com/rubocop-hq/rubocop/issues/9448): Fix an error for `Style/SoleNestedConditional` when using nested `unless` modifier as in single expression. ([@koic][])
```

Comment:
I think a better wording would be "unless modifier with a single expression condition".

Reply:
I updated it again. Thank you for taking time 💦